src/adapters/fabric/e2eUtils.js

```diff
@@ -551,7 +551,8 @@ async function getcontext(channelConfig, clientIdx, txModeFile) {
                     'grpc.keepalive_time_ms' : 360000   // time to wait for ping response, 6 minutes
                     // 'grpc.http2.keepalive_time' : 15
                 }
-            );
+            );*/
+            let eh = channel.newChannelEventHub(peer);
```

Comment:
This call is allowed by the 1.1 SDK, but it's not compatible with a 1.0 Fabric backend.